client/src/pages/login/index.js

```diff
@@ -1,52 +1,135 @@
-import React, { Component } from 'react';
-import { Redirect } from 'react-router-dom';
+import React, {
+  Component
+} from 'react';
+import {
+  Redirect
+} from 'react-router-dom';
+import SweetAlert from 'react-bootstrap-sweetalert';
 
-import { Input, Button } from '../../components';
+import {
+  Input,
+  Button
+} from '../../components';
 import './login.css';
 
 class Login extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      textInputValue: '',
-      isLogin: false
+      textInputValueUsername: '',
+      textInputValuePassword: '',
+      isLogin: false,
+      alert: null
     };
 
-    this.handleTextInputChange = this.handleTextInputChange.bind(this);
+    this.handleUsernameInputChange = this.handleUsernameInputChange.bind(this);
+    this.handlePasswordInputChange = this.handlePasswordInputChange.bind(this);
     this.handleLogin = this.handleLogin.bind(this);
+  }
+
+  errorPassOrUsername() {
+    const getAlert = () => (<SweetAlert
+      warning title = 'Wrong!'
+      confirmBtnBsStyle = 'danger'
+      cancelBtnBsStyle = 'danger'
+      onConfirm = {
+        () => this.hideAlert()
+      } >
+      Invalied Password or Username!
+    </SweetAlert>
+    );
+
+    this.setState({
+      alert: getAlert()
+    });
+  }
 
+  hideAlert() {
+    console.log('Hiding alert...');
+    this.setState({
+      alert: null
+    });
   }
 
   handleLogin(e) {
     e.preventDefault();
-    this.loginStatus();
+    const {
+      textInputValueUsername,
+      textInputValuePassword
+    } = this.state;
+    const data = JSON.stringify({
+      textInputValueUsername,
+      textInputValuePassword
+    });
+    fetch('/login_post', {
+      credentials: 'same-origin',
+      headers: {
+        'content-type': 'application/json'
+      },
+      method: 'POST',
+      body: data
+    })
+      .then(response => response.json(data))
+      .then(response => {
+        console.log('data', response);
+
+        if (response.doneLogin) {
+          this.setState({
+            isLogin: true
+          });
+        } else {
+          this.setState({
+            isLogin: false
+          });
+          this.errorPassOrUsername();
+        }
+      })
+      .catch(err => {
+        console.log('There has been an error ', err);
+      });
   }
 
-  loginStatus() {
+  handleUsernameInputChange(value) {
+    console.log(value);
     this.setState({
-      isLogin: !this.state.isLogin
+      textInputValueUsername: value
     });
   }
 
-  handleTextInputChange(value) {
+  handlePasswordInputChange(value) {
     console.log(value);
-    this.setState({ textInputValue: value });
+    this.setState({
+      textInputValuePassword: value
+    });
   }
+
   render() {
+    console.log(this.state.isLogin);
+
     return (
       <div className='container'>
+        {this.state.alert}
```

Comment:
Instead of storing the alert component in state, you might want to store a boolean (showLoginError) and put the SweetAlert component here in render() if state.showLoginError is true.
The react docs recommend that you don't store components in state: http://web.archive.org/web/20150419023006/http://facebook.github.io/react/docs/interactivity-and-dynamic-uis.html#what-shouldnt-go-in-state